entity-framework/core/what-is-new/ef-core-9.0/breaking-changes.md

```diff
@@ -108,6 +109,26 @@ var newBehavior = context.Entity.Select(x => x.NullableBool.ToString());
 var oldBehavior = context.Entity.Select(x => x.NullableBool == null ? null : x.NullableBool.ToString());
 ```
 
+<a name="shared-framework-dependencies"></a>
+
+### Shared framework dependencies were updated to 9.0.x
+
+#### Old behavior
+
+Apps using the `Microsoft.NET.Sdk.Web` SDK and targetting net8.0 would resolve packages like `Microsoft.Extensions.Caching.Memory`, `Microsoft.Extensions.Configuration.Abstractions`, `Microsoft.Extensions.Logging` and `Microsoft.Extensions.DependencyModel` from the shared framework, so these assemblies wouldn't normally be deployed with the app.
+
+#### New behavior
+
+While EF Core 9.0 still supports net8.0 it now references the 9.0.x versions of `Microsoft.Extensions.Caching.Memory`, `Microsoft.Extensions.Configuration.Abstractions`, `Microsoft.Extensions.Logging` and `Microsoft.Extensions.DependencyModel`. Apps targetting net8.0 will not be able to levarage the shared framework to avoid deploying these assemblies.
```

Comment:
```suggestion
While EF Core 9.0 still supports net8.0, it now references the 9.0.x versions of `Microsoft.Extensions.Caching.Memory`, `Microsoft.Extensions.Configuration.Abstractions`, `Microsoft.Extensions.Logging` and `Microsoft.Extensions.DependencyModel`. Apps targetting net8.0 will not be able to leverage the shared framework to avoid deploying these assemblies.
```